applications/ConvectionDiffusionApplication/python_scripts/convection_diffusion_base_solver.py

```diff
@@ -565,6 +579,9 @@ def _create_linear_solver(self):
         return linear_solver
 
     def _create_builder_and_solver(self):
+        if self.settings["time_integration_method"].GetString() == "explicit":
+            builder_and_solver = KratosMultiphysics.ExplicitBuilder()
+            return builder_and_solver
```

Comment:
Why are you adding this to the base solver if you are creating your own explicit one?

Reply:
Moving to explicit solver.